x-pack/plugin/sql/qa/src/main/resources/conditionals.csv-spec

```diff
@@ -0,0 +1,163 @@
+caseField
+SELECT emp_no, CASE WHEN emp_no - 10000 < 10 THEN 'First 10' ELSE 'Second 10' END as "case" FROM test_emp WHERE emp_no >= 10005
+ORDER BY emp_no LIMIT 10;
+
+ emp_no |   case
```

Comment:
Following this PR - https://github.com/elastic/elasticsearch/pull/41355 - can you change one of these tests to not have an alias?